index.html

```diff
@@ -2610,6 +2619,161 @@ <h2>Security Considerations</h2>
 
   </section>
 
+  <section class="appendix">
+    <h4>Timecode-related metadata</h4>
+    <p><a>DAPT Documents</a> express time as <em>media time</em>,
+      which assumes that there is a reference start time (zero) on a media
+      timeline, and a fixed playrate.
+      An alternative scheme that is used in some workflows is to
+      synchronise media components using timecode,
+      time stamps that are applied, for example to each frame of video.
+    </p>
+    <p>Workflows that create <a>DAPT documents</a> from
+      resources that use timecode need to map the timecode
+      values onto the <a>DAPT document</a>'s timeline.</p>
+    <p>If this mapping is not correct, presentation of the
+      <a>DAPT Document</a> will not be synchronised with the <a>related media object</a>.
+      In order to achieve correct synchronisation in this scenario,
+      the timecode corresponding to the start of the programme needs to be known.</p>
+    <p>If the start of programme timecode is not known,
+      but timecodes corresponding to <a>Script Events</a> are known,
+      it is still possible to construct a <a>DAPT Document</a>,
+      albeit one whose synchronisation with the related media is uncertain.</p>
+    <p>The optional <a>DAPT Origin Timecode</a> and
+      <a>Start of Programme Timecode</a> properties can be used to
+      identify when there is a possible synchronisation error,
+      and to resynchronise the document when all
+      the required information is known.</p>
```

Comment:
If my understanding is correct I would suggest replacing with the following text

> The properties can be used to provide timecodes from the related media object or from any other script format used to produce the DAPT document, and are informational. However, when they are both provided and differ, it is an indication that the DAPT document is not synchronized with the related media object and that processing of the script event begins is needed. To achieve synchronization, the following needs to be done:
> - The difference DAPT Origin Timecode minus the Start of Programme Timecode is computed as "delta". It may be positive or negative.
> - Each Script begin value X shall be changed to X + delta.
> - The DAPT Origin Timecode shall be removed or changed to the Start of Programme timecode value.